unit-tests/log/test-enable-rolling.py

```diff
@@ -0,0 +1,50 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+from rspy import log, test
+import pyrealsense2 as rs
+import common
+
+
+import tempfile, os.path
+temp_dir = tempfile.gettempdir()
+
+
+#############################################################################################
+#
+test.start( f"Rolling logger (max 1M)" )
+
+try:
+    log_filename = os.path.join( temp_dir, 'rolling.log' )
+    rs.log_to_file( rs.log_severity.info, log_filename )
+
+    max_size = 1  # MB
+    rs.enable_rolling_log_file( max_size )
+
+    for i in range( 15000 ):
+        rs.log( rs.log_severity.info, f'debug message {i}' )
+    rs.reset_logger()
+
+    with open( log_filename, "rb" ) as log_file:
+        log_file.seek( 0, 2 )  # 0 bytes from end
+        log_size = log_file.tell()
+    del log_file
+    log.d( f'{log_filename} size: {log_size}' )
+
+    old_filename = log_filename + ".old"
+    with open( old_filename, "rb" ) as old_file:
+        old_file.seek( 0, 2 )  # 0 bytes from end
+        old_size = old_file.tell()
+    del old_file
+    log.d( f'{old_filename} size: {log_size}' )
+
+    max_size_in_bytes = max_size * 1024 * 1024
```

Comment:
Do you understand this test??
I couldn't understand what does it test..

Reply:
It sets the max roll size to 1MB. In reality, that means the files are going to be half that in max size. We're writing enough here to have one rollover, so we'll end up with two files.

Reply:
That's weird no?
why to we split it in half.
OK This test is missing explanation... but that's not for you